lib/init.go

```diff
@@ -0,0 +1,221 @@
+package lib
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/alecthomas/template"
```

Comment:
Is there any reason to use this package instead of text/template?

Reply:
As I understand it's a workaround around whitespace. I dislike the introduction of a separate dependency for such a small thing. What where the other workaround you deemed insufficient?

Reply:
@stp-ip @erbesharat  have no idea why my editor formats itself with this instead of text/template